src/modrover/rover.py

```diff
@@ -503,14 +505,19 @@ def _get_super_coef(
         return super_coef
 
     def _get_super_vcov(
-        self, learner_ids: list[LearnerID], weights: NDArray
+        self,
+        learner_ids: list[LearnerID],
+        weights: NDArray,
+        super_coef: NDArray,
     ) -> NDArray:
         super_vcov = np.zeros((self.num_vars, self.num_vars))
         for learner_id, weight in zip(learner_ids, weights):
+            learner = self.learners[learner_id]
             coef_index = self._get_coef_index(learner_id)
-            super_vcov[np.ix_(coef_index, coef_index)] += (
-                weight * self.learners[learner_id].vcov
+            super_vcov[np.ix_(coef_index, coef_index)] += weight * (
```

Comment:
Just curious what this extra `np.outer` term adds? If you don't mind explaining

Reply:
Hi @davidshaw-uw, the formula is from the law of total (co)variance.
https://en.wikipedia.org/wiki/Law_of_total_covariance
The first term is the expectation of the covariance, and the second term (`np.outer`) is the covariance of the expectation.